installer/templates/phx_assets/package.json

```diff
@@ -11,16 +11,13 @@
     "topbar": "^0.1.4"<% end %>
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "babel-loader": "^8.0.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.4.2",
-    "hard-source-webpack-plugin": "^0.13.1",
-    "mini-css-extract-plugin": "^0.9.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "terser-webpack-plugin": "^2.3.2",
-    "webpack": "4.41.5",
-    "webpack-cli": "^3.3.2"
+    "copy-webpack-plugin": "^9.0.0",
+    "css-loader": "^5.2.6",
+    "css-minimizer-webpack-plugin": "^3.0.0",
+    "expose-loader": "3.0.0",
```

Comment:
`expose-loader` not being used within `webpack.config.js`.

Reply:
I left it in because @chrismccord and I were going back and forth on if the `Phoenix` constant should be exposed. He fell on the side that it wasn't necessary but I wanted to give the community a chance to provide feedback if for some reason a need for that constant is being overlooked.

Reply:
To be clear, the `Phoenix` constant needs to be on window when the built js is loaded, for folks not using an asset builder and for the live reload project. If we can ditch `expose-loader` and do this directly then great, but we need to make sure it is exposed from the bundle.

Reply:
It could *technically* be done without `expose-loader` but I think it s going to be some crazy rule writing. I would vote to keep the dependency to do this.

Reply:
To be clear, keeping it on the internal webpack config, not the one in the installer template

Reply:
works for me